### ISSUE TITLE
Color Picker don´t take account light/darken values

### ISSUE THREAD

**Issue by DanielMartinezLara** [nvrmind/IssueTracker#10]
If you use the light/darken brush the Color Picker don´t take account this light/darken values

See the gif for more info

![color picker bug](https://user-images.githubusercontent.com/35196303/34836675-e628e97e-f6f8-11e7-9aff-8f87fb206e8c.gif)

Daniel

**Comment by daseyb** [comment]
True, that's definitely something where you want a "What you see is what you get" results.
